docs/release-announcement.md

```diff
@@ -1,11 +1,17 @@
 # Release announcement
 
-## Azure Industrial IoT Platform 2.8 Release
+## Azure Industrial IoT Platform Release 2.8.1
+We are pleased to announce the release of version 2.8.1 of our Industrial IoT Platform components as the first patch update of the 2.8 Long-Term Support (LTS) release. This release contains important security updates, bugfixes and performance optimizations.
 
-We are pleased to announce the release of version 2.8.0 of our Industrial IoT Platform as well as the declaration of Long-Term Support (LTS) for this version.
+> IMPORTANT
+> Please note that OPC Publisher 2.8.1 is not backwards compatible with version 2.5.x. This will be addressed in the next patch release.
```

Comment:
Please remove `This will be addressed in the next patch release.`. Even though this is our plan at the moment, that might change and so should not commit to it in release announcements.